### PR TITLE
tracing - expose jaeger collector endpoint

### PR DESCRIPTION
**Pull request by nwest1**
- Allows user to specify `jaeger_collector` endpoint that overrides `agent_address` (based on client functionality)
- Adds docker-compose with jaeger ports exposed that work with current benthos configurations (agent and collector)

### CODE REVIEW

File: lib/tracer/jaeger.go [hunk 1]
```diff
@@ -20,7 +20,9 @@ func init() {
 		Summary: `
 Send spans to a [Jaeger](https://www.jaegertracing.io/) agent.`,
 		FieldSpecs: docs.FieldSpecs{
-			docs.FieldCommon("agent_address", "The address of a Jaeger agent to send tracing events to."),
+			docs.FieldCommon("agent_address", "The address of a Jaeger agent to send tracing events to.", "jaeger-agent:6831"),
+			docs.FieldCommon("collector_address", "The address of a Jaeger collector to send tracing events to. If set, this will override `agent_address`.",
```
Comment (Jeffail) [inline lib/tracer/jaeger.go:24]: Looks like this field accepts full URLs so it should be called `collector_url` in order to fit in with other Benthos fields.

**Review by Jeffail — COMMENTED** [review]
Hey @nwest1, looks good, left one comment as I think the field name ought to be changed.

**Comment by nwest1** [comment]
Thanks again @Jeffail !

**Comment by Jeffail** [comment]
Looks good, nice one @nwest1!

**Pull request merged**